src/app.jsx

```diff
@@ -1,3 +1,9 @@
+/* eslint no-console: ["error", { allow: ["log", "warn", "info", "error"] }] */
```

Comment:
here I prefer to disable linter issues only because this file will be gone in coming prs.